Virtusize/Sources/Internal/Models/VirtusizeGetSizeParams.swift

```diff
@@ -49,9 +49,6 @@ internal struct VirtusizeGetSizeParams: Codable {
         userBodyProfile: VirtusizeUserBodyProfile?
 
     ) {
-
-   /// Chirag : Commented modelInfo not getting proper value  but its optional
-   /// Chirag: Gender value getting from userBodyProfile, not getting from storeProduct.storeProductMeta?.additionalInfo
```

Comment:
Removed personal comments.
If modelInfo is not available, it needs to be `null` instead of '{}'.

Reply:
Yes. Thanks !